python/ray/rllib/bc/experience_dataset.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
```

Comment:
We will need to extend the functionality of this class to work on multiple machines.

Reply:
yea, it's currently loading the dataset from a specified path; we may want to ray.put the dataset, would that be a problem with really big datasets (say up to several 10GB)? what's the alternative?

Reply:
depends on how much memory you allocate for your instance, but you could always partition the dataset so that the object store acts more as a buffer. I think `ray.put` should be fine.

Reply:
Put could be hazardous if the dataset gets evicted at some point. Maybe for now just document that the files should be available on each machine.